utils/src/treev2/quickPickWizard/FindByIdQuickPickStep.ts

```diff
@@ -15,15 +16,14 @@ interface FindByIdQuickPickOptions extends SkipIfOneQuickPickOptions {
 
 export class FindByIdQuickPickStep<TContext extends types.QuickPickWizardContext> extends GenericQuickPickStep<TContext, FindByIdQuickPickOptions> {
     public constructor(tdp: vscode.TreeDataProvider<unknown>, options: FindByIdQuickPickOptions) {
-        super(
-            tdp,
-            {
-                ...options,
-                skipIfOne: true, // Find-by-id is always skip-if-one
-            }
-        );
+        super(tdp, {
+            ...options,
+            skipIfOne: true, // Find-by-id is always skip-if-one
+        });
     }
 
+    readonly pickFilter: PickFilter = new FindByIdPickFilter(this.pickOptions);

```

Comment:
Can we keep `public` here and all the rest of the places (or `protected`, `private` as needed)?

Reply:
Yes. Is the reasoning just to be more explicit?

Reply:
Yep. Explicit and consistent.